### PR TITLE
Fix vent critters spawn

### PR DESCRIPTION
**Pull request by Slava0135**
## About the PR

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: vent critters can no longer spawn on CentCom
- tweak: number of vent critters is now proportional to number of spawn points

### CODE REVIEW

**Comment by Slava0135** [comment]
I tested this on Box and got 10 spiders on average (5% chance)

**Comment by Slava0135** [comment]
I need to fix prototypes i guess

**Comment by Slava0135** [comment]
Should check if there any valid locations when spawning

File: Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs [hunk 1]
```diff
@@ -8,4 +8,10 @@ public sealed class VentCrittersRuleComponent : Component
 {
     [DataField("entries")]
     public List<EntitySpawnEntry> Entries = new();
+
+    /// <summary>
+    /// At least one special entry is guaranteed to spawn
+    /// </summary>
+    [DataField("specialEntries")]
+    public List<EntitySpawnEntry> SpecialEntries = new();
```
Comment (metalgearsloth) [inline Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs:16]: Isn't this just a separate orGroup.

Comment (Slava0135) [reply Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs:16]: guaranteed spawn is per station, not vent. I think orGroup ALWAYS spawns at least one entry from group, i tried it and got like 200 spawns on Box.

**Pull request merged**